agir/groups/components/messages/PromoMessageModal.js

```diff
@@ -0,0 +1,283 @@
+import promo4 from "./promo-message4.svg";
+
+const CloseButton = styled.button``;
+
+const StyledModalContent = styled.div`
+  position: relative;
+  text-align: center;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: flex-start;
+  max-width: 600px;
+  padding: 0 0 36px;
+  margin: 60px auto 0;
+  box-shadow: ${style.elaborateShadow};
+  border-radius: 8px;
+  background-color: ${style.white};
+  overflow-x: hidden;
+  overflow-y: auto;
+
+  @media (max-width: ${style.collapse}px) {
+    margin-top: 20px;
+    max-width: calc(100% - 40px);
+    padding: 0 0 1.5rem;
+  }
+
+  & > * {
+    margin: 0;
+    padding: 0 3.375rem;
+
+    @media (max-width: ${style.collapse}px) {
+      padding: 0 1.5rem;
+    }
+  }
+
+  ${CloseButton} {
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    padding: 0;
+    color: ${style.black1000};
+    z-index: 1;
+    background-color: transparent;
+    border: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+  }
+
+  header {
+    position: relative;
+    width: 100%;
+    height: 177px;
+    background-color: ${style.secondary500};
+    background-position: bottom center;
+    background-repeat: no-repeat;
+    margin-bottom: 56px;
+
+    @media (max-width: ${style.collapse}px) {
+      height: 100px;
+      background-size: 160px auto;
+    }
+
+    &::after {
+      content: "";
+      display: block;
+      background-color: transparent;
+      background-repeat: no-repeat;
+      background-size: cover;
+      width: 100px;
+      height: 100px;
+      position: absolute;
+      bottom: 0;
+      left: 50%;
+      transform: translate3d(-50%, 50%, 0);
+    }
+  }
+`;
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 2rem;
+  overflow-y: auto;
+  max-width: 600px;
+  width: 100%;
+
+  p {
+    margin: 30px 0;
+    font-size: 18px;
+    text-align: center;
+  }
+
+  img {
+    user-select: none;
+  }
+
+  @media (max-width: ${style.collapse}px) {
+    padding: 1rem;
+    img {
+      width: 150px;
+    }
+  }
+`;
+
+const StyledLabel = styled.div`
+  color: ${style.redNSP};
+`;
+
+const Title = styled.div`
+  margin: 0.5rem 0;
+  font-size: 1.5rem;
+  font-weight: 700;
+`;
+
+const Arrow = styled.div`
+  border: 1px solid #dfdfdf;
+  border-radius: 100px;
+  display: inline-flex;
+  padding: 0.5rem;
+  cursor: pointer;
+  user-select: none;
+`;
+
+const Mark = styled.span`
+  width: 0.5rem;
+  height: 0.5rem;
+  margin: 0.188rem;
+  display: inline-block;
+  border-radius: 2rem;
+  cursor: pointer;
+  transition: background-color 0.5s ease-in-out;
+  background-color: ${(props) =>
+    props.$active ? style.black700 : style.black200};
+`;
+
+const items = [
+  {
+    img: promo1,
+    content: (
+      <p>
+        <b>Lancez une première conversation dans votre groupe !</b>
```

Comment:
```suggestion
        <strong>Lancez une première conversation dans votre groupe !</strong>
```
On essaie d'utiliser des éléments `strong` plutôt que des `b`, c'est mieux d'un point de vue sémantique et d'accessibilité